package.json

```diff
@@ -13,7 +13,7 @@
     "build": "webpack -p",
     "transpile": "cross-env BABEL_DISABLE_CACHE=1 babel src --out-dir lib",
     "start": "cross-env NODE_ENV=development webpack-dev-server -d --inline --hot",
-    "prepublish": "npm run build && npm run transpile"
+    "prepublishOnly": "npm run build && npm run transpile"
```

Comment:
From the commit message:

> Note that in CI (Travis and AppVeyor), "npm test" will
run "npm run build", so this scenario is still "tested".

I wholeheartedly support this change, although if I'm not mistaken, the `transpile` part is not run. If we switch to `prepublishOnly`, we should either add `transpile` to `build` or to `test`, or maybe both btw (is there a script name for that?)

Reply:
Isn't transpiling already done when compiling? I mean, Babel runs as part of the Webpack build; it does not output ES5 to disk, but this is mostly equivalent in all other aspects, right?

Reply:
I guess it is, you're right. Actually, I thought we needed the `lib/` folder for unit tests, but actually sources are compiled on the fly by mocha apparently. I think I mixed up with the old version of integration tests, before we used puppeeter.